### PR TITLE
docs: windows trouble shoot on docker mode

### PR DESCRIPTION
**Pull request by ThibaultFy**
Signed-off-by: ThibaultFy <50656860+ThibaultFy@users.noreply.github.com>

### CODE REVIEW

File: docs/source/substrafl_doc/substrafl_overview.rst [hunk 1]
```diff
@@ -22,8 +22,10 @@ Installation
 
 .. _installation:
 
-Substrafl and Substra are compatible with Python versions 3.8, 3.9 and 3.10 on both MacOS and Linux. For Windows users you can use the
-`Windows Subsystem for Linux <https://docs.microsoft.com/en-us/windows/wsl/about>`_.
+Substrafl and Substra are compatible with Python versions 3.8, 3.9 and 3.10 on both Windows, MacOS and Linux.
```
Comment (guilhem-barthes) [inline docs/source/substrafl_doc/substrafl_overview.rst:25]: Both refers to two things 
```suggestion
Substrafl and Substra are compatible with Python versions 3.8, 3.9 and 3.10 on Windows, MacOS and Linux.
```

File: docs/source/documentation/debug.rst [hunk 2]
```diff
@@ -50,6 +50,14 @@ Two local modes are available:
 
 The subprocess mode is much faster than the Docker mode, but does not test that the Dockerfiles of the assets are valid, and may fail if advanced COPY or ADD commands are used in the Dockerfile. Therefore, the Docker mode is closer to the remote mode than the subprocess mode. It is recommended to run your experiment locally in subprocess mode and when it is ready, then test it with the Docker mode.
 
+.. note::
+    For the **Docker mode on Windows**, follow these `instructions <https://docs.docker.com/desktop/install/windows-install/>`__ to install Docker desktop.
+
+    Once installed:
+
+        * In ``Settings`` -> ``General``, check the ``Expose daemon on tcp://localhost:2375 without TLS`` option.
+        * Set the environment variable ``DOCKER_HOST`` to  ``tcp://localhost:2375``
```
Comment (RomainGoussault) [inline docs/source/documentation/debug.rst:59]: Why do we need to do that?

Comment (RomainGoussault) [reply docs/source/documentation/debug.rst:59]: And should write the reason here?

Comment (ThibaultFy) [reply docs/source/documentation/debug.rst:59]: From what I found, by default, you don't have access to the dcker daemon on windows. Either you do this, either you configure your docker to keep the TLS activated.

Comment (RomainGoussault) [reply docs/source/documentation/debug.rst:59]: Ok that's fine.
Let's dig into that later, at least we have a way of making it work!

**Review by RomainGoussault — APPROVED** [review]

**Pull request merged**